cdap-proto/src/main/java/co/cask/cdap/proto/metadata/MetadataSearchTargetType.java

```diff
@@ -15,9 +15,12 @@
  */
 package co.cask.cdap.proto.metadata;
 
+import co.cask.cdap.api.annotation.Beta;
+
 /**
  * Supported types for metadata search.
  */
+@Beta
```

Comment:
Since there is no backwards incompatible change I think this is okay. Spoke to @anew too, and he agrees.